ledger-core/v2/runner/runner.go

```diff
@@ -142,14 +143,67 @@ func generateCallContext(
 		// should be the same as request.Object
 		res.Callee = oDesc.HeadRef()
 	} else {
-		res.Callee = &execution.Request.Caller
+		res.Callee = &execution.Object
 	}
 
 	return res
 }
 
-func (r *DefaultService) executeMethod(_ context.Context, _ uuid.UUID, _ *executionEventSink) {
-	panic(throw.NotImplemented())
+func (r *DefaultService) executeMethod(ctx context.Context, id uuid.UUID, eventSink *executionEventSink) {
```

Comment:
Might it's better, that 'executeMethod' returns ( res, error ). 
In the place where you cal 'executeMethod', you check error, and use 'eventSink' in appropriate way.
What for? 'eventSink' will be used in one place and it will decrease probability that I forget to 'eventSink.ErrorWrapped' in case of occurrence of any other error in 'executeMethod'